tox.ini

```diff
@@ -3,48 +3,45 @@ minversion = 3.18.0
 envlist =
     lint
     docs
+    py{36,37,38,39}-{stable,devel}
     packaging
-    py{36,37,38,39}
-    py{36,37,38,39}-{devel}
 
 # do not enable skip missing to avoid CI false positives
-skip_missing_interpreters = False
-isolated_build = True
+skip_missing_interpreters = false
```

Comment:
That is not an YAML file, if I remember well the correct values for tox ini are with uppercase.